src/main/java/com/minecolonies/coremod/client/gui/containers/WindowFurnaceCrafting.java

```diff
@@ -100,7 +101,7 @@ public void onPress(final Button button)
 
                 if (!ItemStackUtils.isEmpty(primaryOutput))
                 {
-                    Network.getNetwork().sendToServer(new AddRemoveRecipeMessage(building, input, 1, primaryOutput, false));
+                    Network.getNetwork().sendToServer(new AddRemoveRecipeMessage(building, input, 1, primaryOutput, null, false));
                 }
```

Comment:
why not an empty list?

Reply:
We could, but it will become an empty list either way due to the recipestorage constructor.

Reply:
Went ahead and did an empty list.